### ISSUE TITLE
how to improve tts result

### ISSUE THREAD

**Issue by freecui** [begeekmyfriend/tacotron2#7]
![step-140000-mel-spectrogram](https://user-images.githubusercontent.com/8288975/70884336-bcf59c80-2010-11ea-903a-3eb2a0664af3.png)
![step-140000-align](https://user-images.githubusercontent.com/8288975/70884354-c4b54100-2010-11ea-8925-7e9fade3ab69.png)
this is my synthesizer result, is that ok ?

[result.zip](https://github.com/begeekmyfriend/tacotron2/files/3966363/result.zip)

this is my vocoder result, and you can hear the result audio (130k_steps_5_gen_batched_target8000_overlap400) is not good, it seems noise, I don't know how to solve it?

**Comment by freecui** [comment]
I didn't trim leading and trailing silence on the training data

**Comment by begeekmyfriend** [comment]
Did you use `preprocess.py` for your training data? And `script/inference_tacotron2.sh` for G&L evaluation? The mel spectrograms for this project range from [-4, 4].
```shell
python preprocess.py
bash script/inference_tacotron2.sh
```

**Comment by freecui** [comment]
do you mean normalize? yes, I did normalize, and  max_abs_value=4 ; but I did that according to the project https://github.com/CorentinJ/Real-Time-Voice-Cloning, and I just modify the symbols.py and 'cleaners' like your project. Is there any problem on preprocessing the audio with Real-Time-Voice-Cloning for chinese? Of cause , I did not use him algining the text and audio

**Comment by freecui** [comment]
I don't think my trained audios are very  standard mandarin, do you think the bad result is related audios ?

**Comment by begeekmyfriend** [comment]
130k is too short for wavernn training, please wait till 600k step and be patient.

**Comment by freecui** [comment]
ok, I will continue training wavernn. what your loss on wavernn?  my wavernn loss is more than 2 on 130k steps

**Comment by freecui** [comment]
my synthesizer loss is 3.4+, and I found the loss didn't decline.

**Comment by freecui** [comment]
![微信图片_20191216175500](https://user-images.githubusercontent.com/8288975/70897397-795d5b80-202d-11ea-9187-a7b467e6749f.jpg)

**Comment by begeekmyfriend** [comment]
Different mel spectrograms have different loss values. Typically the loss value would be steady at ~600k step.
